src/i_winmusic.c

```diff
@@ -230,73 +230,77 @@ static void StreamOut(void)
     }
 }
 
-static void SendShortMsg(int time, int status, int channel, int param1, int param2)
+static void SendShortMsg(unsigned int delta_time, byte status, byte channel,
```

Comment:
Are you sure this is safe? `MAKE_EVT` uses bitshifts on these without upcast. I don't remember the C implicit casting rules in that case.

Reply:
The bytes are promoted to ints before bitshifting.

Reply:
Fixed by https://github.com/chocolate-doom/chocolate-doom/pull/1598/commits/74cd516fa5c9a90e73d297d41c30f62b1f2da48f